src/features/commands.ts

```diff
@@ -392,6 +392,24 @@ To integrate it into your editor: https://prettier.io/docs/en/editors.html`,
         }
       });
     }
+  },
+  {
+    words: [`!gender`],
+    help: `reminds users to use gender-neutral language`,
+    handleMessage: msg => {
+      msg.channel.send({
+        embed: {
+          title: "Please use gender neutral language by default",
+          type: "rich",
+          description: `Unless someone has made their gender known, please use gender neutral language.
```

Comment:
A gender identity does not necessarily always use the same pronouns, so I think it's better to word this in terms of gender pronouns rather than just genders.
```suggestion
          description: `Unless someone has made their gender pronouns known, please use gender neutral language.
```